tests/components/datadog/test_init.py

```diff
@@ -145,11 +134,11 @@ def test_state_changed(self, mock_connection, mock_client):
             )
             handler_method(mock.MagicMock(data={"new_state": state}))
 
-            assert mock_client.gauge.call_count == 5
+            assert mock_statsd.gauge.call_count == 5
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6356951)